src/cluster/k8sClientHandler.go

```diff
@@ -475,3 +475,25 @@ func GetNodesFromK8sClient() (*v1.NodeList, error) {
 	}
 	return nodeList, nil
 }
+
+func GetKubearmorRelayURL() string {
+	var namespace string
+	client := ConnectK8sClient()
+	if client == nil {
+		return ""
+	}
+
+	// get kubearmor pods from k8s api client
+	pods, err := client.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
+		LabelSelector: "kubearmor-app",
+	})
```

Comment:
Let's directly fetch the relay pod. 

Ref https://github.com/kubearmor/KubeArmor/blob/d6db731c7e4383faa7cd0518b750703dc017e71e/deployments/get/objects.go#L110

```suggestion
	pods, err := client.CoreV1().Pods("").List(context.Background(), metav1.ListOptions{
		LabelSelector: "kubearmor-app=kubearmor-relay",
	})
```